### DISCUSSION TITLE
Problem with OrturLasermaster 15W and LaserGrbl

### DISCUSSION THREAD

**Discussion by MCSpider65** [arkypita/LaserGRBL#1059] [Q&A]
Hey Community,

I hope you can help me. I bought a Ortur Lasermaster 15 W. It works fantastic on the first day, on second day (after restart the computer) I got a "Soft limit" Error.
I did a reset, new installing all drivers and software, but nothing works.
Hope somebody can help me, otherwise it was waste money and I put it away. I don´t know what to do.

I put some screenshots helping to show the problem.

![AfterAddingFile](https://user-images.githubusercontent.com/67433411/85767786-a5fada00-b718-11ea-9165-f4bc69c6fa77.jpg)
![AfterStartEngraving](https://user-images.githubusercontent.com/67433411/85767788-a6937080-b718-11ea-8a97-ccf5f5d89f79.jpg)
![ProblemMessage](https://user-images.githubusercontent.com/67433411/85767789-a72c0700-b718-11ea-8e20-effa8010dc74.jpg)
![SoftLimitError](https://user-images.githubusercontent.com/67433411/85767791-a72c0700-b718-11ea-86f3-c50e0657b465.jpg)

**Comment by arkypita** [comment]
The soft-limit error is generated by the hardware to prevent you from moving the laser beyond the physical limits of the axes.
Each command sent (jog or marking) is controlled according to the frame size configured in grbl configuration.

![image](https://user-images.githubusercontent.com/8782035/85833943-d2ebd300-b792-11ea-81b6-5e4c8fad8222.png)

**Comment by arkypita** [comment]
You should check travel size you set in $130-$131 and/or you can disable soft limit by set $20 to 0
